.github/workflows/lint.yml

```diff
@@ -35,3 +35,5 @@ jobs:
         run: tox -e mypy
       - name: Run isort import order checker (https://pycqa.github.io/isort/)
         run: tox -e isort -- --check
+      - name: Run pylint Python code static checker (https://www.pylint.org/)
+        run: tox -e pylint
```

Comment:
Can we also add this to pre-commit? 
https://pylint.pycqa.org/en/latest/user_guide/pre-commit-integration.html

Reply:
Done! 😊